components/director/internal/domain/label/repository.go

```diff
@@ -173,6 +173,31 @@ func (r *repository) DeleteByKey(ctx context.Context, tenant string, key string)
 
 	return nil
 }
+func (r *repository) GetRuntimeScenariosWhereRuntimesLabelsMatchSelector(ctx context.Context, tenantID, selectorKey, selectorValue string) ([]model.Label, error) {
```

Comment:
What's the difference between this method and `GetRuntimeScenariosWhereLabelsMatchSelector`?

Reply:
It's leftover. I will remove it.